src/css/main.css

```diff
@@ -20,3 +20,9 @@
 .clip-bottom-8 {
   clip-path: polygon(0 0, 100% 0, 100% calc(100% - 8px), 0 calc(100% - 8px));
 }
+
+.hr {
+  color: #DEDEE2;
```

Comment:
Could we add another color variable to the tailwind.config.js for this? Or maybe just re-use `dark-gray` with some opacity?